ingc_geonode_theme/templates/index.html

```diff
@@ -44,7 +44,7 @@ <h1>{% trans "What is it all about?" %}</h1>
             </tr>
             <tr>
                 <td colspan="2">
-                    <p style="font-size: 1.5em;">
+                    <p>
                         {% trans "This platform is part of an initiative where the Government of Mozambique and the World Bank have been promoting the use of open geospatial data for government disaster risk management through the Open Data for Resilience Initiative to provide technical assistance to INGC" %}
```

Comment:
Is this intended? You need to put it on blocktrans, so the translator get a complete paragraph.